src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -612,6 +612,9 @@ private void testPublishAction() throws Exception {
         feedbackPage.clickAndCancel(feedbackPage.getPublishLink(courseId, sessionName));
         assertFalse(BackDoor.getFeedbackSession(courseId, sessionName).isPublished());
 
+        // Test that the resend published link button doesn't exist
+        feedbackPage.verifyResendPublishedEmailButtonDoesNotExist(courseId, sessionName);
```

Comment:
How about refactoring the `verifyResendPublishedEmailButtonDoesNotExist()` function to `checkIfResendPublishedEmailButtonExists` and make it return true/false depending on the exception. Then you can also check later if button exists using `assertTrue/False()` functions